### PR TITLE
hide output in quite mode

### PR DESCRIPTION
**Pull request by whyte624**
The output is hidden now in quite mode. Output will appear only with -v option.
Remove unnecessary imports.

### CODE REVIEW

File: src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php [hunk 1]
```diff
@@ -46,23 +44,33 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $source = $input->getArgument('source');
-
-        $bundlePath = $this->getApplication()->getKernel()->getBundle('MaxmindGeoipBundle')->getPath();
+        /** @var \Symfony\Bundle\FrameworkBundle\Console\Application $application */
```
Comment (idciconsulting) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: Why this comment ?

File: src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php [hunk 2]
```diff
@@ -46,23 +44,33 @@ protected function configure()
...
+        $application = $this->getApplication();
+        $bundlePath = $application->getKernel()->getBundle('MaxmindGeoipBundle')->getPath();
         $dataDir = sprintf('%s', $bundlePath.'/../../../../data/');
         $filename = basename($source);
         $destination = sprintf('%s/%s', $dataDir, $filename);
-        $output->writeln(sprintf('Start downloading %s', $source));
-        $output->writeln('...');
+        $verbose = $output instanceof Output && $output->isVerbose();
```
Comment (idciconsulting) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: Why this test "$output instanceof Output" ?

File: src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php (hunk 1 above)
Comment (whyte624) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: `$this->getApplication()` returns Symfony\Component\Console\Application which does not have method getKernel. With this annotation you'll see `getKernel()` method.

File: src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php [hunk 3]
```diff
@@ -46,23 +44,33 @@ protected function configure()
...
+        if ($verbose) {
+            $output->writeln(sprintf('Start downloading %s', $source));
+            $output->writeln('...');
+        }
         if (!copy($source, $destination)) {
-            $output->writeln('<error>Error during file download occured</error>');
+            if ($verbose) {
+                $output->writeln('<error>Error during file download occured</error>');
```
Comment (idciconsulting) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: Must be in verbose mode to see error ?

File: src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php (hunk 1 above)
Comment (idciconsulting) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: Why create a new variable $application ?

Comment (whyte624) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: To add the annotation.

File: src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php (hunk 3 above)
Comment (whyte624) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: Good point. Let's show the message.

File: src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php (hunk 1 above)
Comment (idciconsulting) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: This is an editor (ide) specificity.

Comment (whyte624) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: No. It is php annotation. And yes, this annotation is used by IDEs.

Comment (idciconsulting) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: So you reserved memory for a single var, used once, only to have the completion in your editor ?

Comment (whyte624) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: Not only for completion. But to point out which class is used and where getKernel method appears from.

Comment (idciconsulting) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: Yes but you get this information with the documentation about "$this->getApplication();". If you wish to reused the '$application' variable why not, but here it's not a good idea.

"GOOD CODE IS LIKE A GOOD JOKE, IT NEEDS NO EXPLANATIONS" ;)

Comment (whyte624) [inline src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php]: Updated.

**Comment by whyte624** [comment]
Tag it, please! Thanks!

**Pull request merged**